game_eggs/minecraft/README.md

```diff
@@ -41,10 +41,10 @@ It’s set in infinitely-generated worlds of wide open terrain - icy mountains,
 * [Cross Platform](proxy/cross_platform)
   * [GeyserMC](proxy/cross_platform/geyser)
   * [Waterdog](proxy/cross_platform/waterdog)
-* [Java](proxy/java)
-  * [FlameCord](proxy/java/flamecord)
-  * [Travertine](proxy/java/travertine)
-  * [TyphoonLimbo](proxy/java/typhoonlimbo)
-  * [Velocity](proxy/java/velocity)
-  * [VIAaas](proxy/java/viaaas)
-  * [Waterfall](proxy/java/waterfall)
+* [Java](proxyjava)
```

Comment:
These definitely have a / between proxy and java

Reply:
Sorry about that, I don't know how that happened. I'll get that fixed.

Reply:
I've double checked all links to ensure they're going to the correct location!